### ISSUE TITLE
Walkthrough needed

### ISSUE THREAD

**Issue by gwct** [phyloacc/PhyloAcc#52]
One thing on my radar is to provide a walkthrough from how to get from whole genome alignments or other data to the PhyloAcc input: predicted conserved non-coding regions and neutral rates/branch lengths.

This will likely appear on [the website](https://phyloacc.github.io/walkthrough.html) sometime this fall (2023) as well as in some other contexts.

**Comment by Asrix** [comment]
This would still be helpful. Going from Cactus output to actually running phyloAcc.

**Comment by gwct** [comment]
@Asrix yes this is my white whale currently. Everytime I work on it I seem to run into another hitch, or a more pressing project comes up. I'm very hopeful though to have this soon.

In the meantime, if you have any specific questions, I'd be happy to help out as much as I can.

**Comment by gwct** [comment]
Addressed (finally) in https://github.com/phyloacc/phyloacc.github.io/pull/1.

See: https://phyloacc.github.io/walkthrough.html

Will continue to improve.
